src/test/java/org/assertj/assertions/generator/util/ClassUtilTest.java

```diff
@@ -229,6 +237,11 @@ public void testGetSimpleNameWithOuterClass_notNestedClass() throws Exception {
   }
 
   @Test
+  public void testGetSimpleNameOuterClass_notNestedClass() throws Exception {
```

Comment:
rename the test using an english sentence to express what the test verifies then put `_` instead of space.

Note: starting test method is redundant since we already have an `@Test` annotation